app/pods/publications/index/template.hbs

```diff
@@ -21,297 +21,293 @@
 
       <WebComponents::AuToolbar::Item>
         <WebComponents::AuButton data-test-publication-header-button-new @icon="add" @skin="primary" {{on "click"
-          (fn this.showPublicationModal)}}>
+                                                                                                          (fn this.showPublicationModal)}}>
           {{t "publications-new"}}
         </WebComponents::AuButton>
       </WebComponents::AuToolbar::Item>
     </WebComponents::AuToolbar::Group>
   </WebComponents::AuToolbar>
 </WebComponents::AuNavbar>
-<div class="auk-scroll-wrapper__body auk-u-m-0">
-  <div class="auk-u-m-4">
-    <WebComponents::AuButton @icon="settings" @skin="borderless" data-test-publication-button-filter-tables
-      {{on "click" this.openColumnDisplayOptionsModal}}
-    />
-    <DataTable
-      @content={{this.model}}
-      @page={{this.page}}
-      @noDataMessage={{t "no-results-found"}}
-      @size={{this.size}}
-      @sort={{this.sort}}
-      @isLoading={{this.isLoadingModel}}
-      @onClickRow={{this.navigateToPublication}}
```

Comment:
deze onclickrow heeft geen refactor gekregen, en de rijen zijn nu niet meer klikbaar. gebruik die `this.navigateToPublication`

Reply:
moet niet, express eruit gelaten